smithy.yaml

```diff
@@ -1,12 +0,0 @@
-project: dart
```

Comment:
i'm only hesitant on this if we have to revert back to the old build system

Reply:
Ok, it doesn’t hurt to leave it for now, I’ll revert this removal.